omod/src/test/java/org/openmrs/module/dataintegrity/web/rest/resource/IntegrityResultPatientProgramResourceTest.java

```diff
@@ -0,0 +1,52 @@
+package org.openmrs.module.dataintegrity.web.rest.resource;
+
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.Patient;
+import org.openmrs.PatientProgram;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.dataintegrity.web.rest.controller.DataIntegrityRestController;
+import org.openmrs.module.webservices.rest.web.RequestContext;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.api.RestService;
+import org.openmrs.module.webservices.rest.web.resource.impl.NeedsPaging;
+import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+public class IntegrityResultPatientProgramResourceTest extends BaseModuleWebContextSensitiveTest {
+
+    private static final String DATAINTEGRITY_DATASET_XML = "org/openmrs/module/dataintegrity/include/DataIntegrityServiceTest-rules.xml";
+    private static Logger logger = Logger.getLogger("logger");
```

Comment:
This is the test class, then why we want the Logger here?

Reply:
@suthagar23 , I'm was testing the unit test and that's why I needed the logger.

Reply:
Shall we use the logger while test cases running?